test/e2e/data/infrastructure-aws/withoutclusterclass/kustomize_sources/addons/csi/data/aws-ebs-csi-external.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: aws-secret
   namespace: kube-system
 stringData:
-  key_id: ${AWS_ACCESS_KEY_ID}
```

Comment:
I remember there was a discussion around this, but it's worth noting why these have been deleted?

Reply:
Meh. I can't find it. I swear someone explained why that was removed. :D

Reply:
I guess the test failures are because of that (except for MD remediation which is already failing)
@wyike could you please revert this change, or maybe find a solution such that other tests passes.

Reply:
comment here: https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/4147#discussion_r1185647464

Reply:
It should be removed. After it is removed, the csi add on will fallback to workload cluster control plane instance role to get credentials from metadata service.